apps/iris/src/loader/postgres.go

```
@@ -58,41 +59,137 @@ func parseDatabaseURL(databaseURL string) (string, error) {
+	if err != nil {
+		p.logger.Log(logger.ERROR, fmt.Sprintf("sql.save.failed stage=begin_tx err=%v", err))
+		return fmt.Errorf("failed to begin transaction: %w", err)
+	}
+	defer tx.Rollback() //nolint:errcheck
+
+	stmt, err := tx.PrepareContext(p.ctx, insertQuery)
+	if err != nil {
+		p.logger.Log(logger.ERROR, fmt.Sprintf("sql.save.failed stage=prepare err=%v", err))
+		return fmt.Errorf("failed to prepare statement: %w", err)
+	}
+	defer stmt.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `stmt.Close` is not checked

(errcheck)